glitter_events/templatetags/events_tags.py

```diff
@@ -7,11 +9,15 @@
 
 
 @register.assignment_tag
-def get_upcoming_events(count=5, category=None):
+def get_upcoming_events(count=5, category='', tags=''):
     event_list = Event.objects.published().filter(start__gt=timezone.now())
 
     # Optional filter by category
-    if category is not None:
+    if category.strip() is not '':
```

Comment:
Don't know, because it just never occurred to me to do it that way? I am happy to expect that:
`if not category.strip():`
is better than
`if category.strip() is not '':`
personally I am not convinced either way.